src/TodoList.js

```diff
@@ -1,11 +1,11 @@
 import React from "react";
 import TodoListItem from "./TodoListItem";
 
-function TodoList ({ todoList }) {
+function TodoList({ todoList, onRemoveTodo }) {
     return (
         <ul>
             {todoList.map(({ id, title }) => (
-                <TodoListItem key={id} todoTitle={title} />
+                <TodoListItem key={id} todoTitle={title} onRemoveTodo={() => onRemoveTodo(id)} />
```

Comment:
Sorry if I suggested destructuring here, if I did that was when the `id` wasn't being used in the `TodoListItem`. Since `onRemoveTodo` needs the `id` it would be better practice to pass `todo` and `onRemoveTodo` down as props. I'll explain more in another comment

```suggestion
                <TodoListItem key={id} todo={todo} onRemoveTodo={onRemoveTodo} />
```